themes/icore-open/fonts/fonts.scss

```diff
@@ -5,40 +5,36 @@
 /* IBM Plex Sans */
 /* Regular */
 @font-face {
-  font-family: "IBM Plex Sans", sans-serif;
+  font-family: "IBM Plex Sans";
   font-weight: normal;
   font-style: normal;
   font-display: var(--text-body-font-display, swap);
-  src: url("#{variables.$font-path}/IBMPlexSans/IBMPlexSans-Regular.ttf")
-    format("ttf");
+  src: url("#{variables.$font-path}/IBMPlexSans/IBMPlexSans-Regular.ttf");
 }
 
 @font-face {
-  font-family: "IBM Plex Sans", sans-serif;
+  font-family: "IBM Plex Sans";
   font-weight: normal;
   font-style: italic;
   font-display: var(--text-body-font-display, swap);
-  src: url("#{variables.$font-path}/IBMPlexSans/IBMPlexSans-Italic.ttf")
-    format("ttf");
+  src: url("#{variables.$font-path}/IBMPlexSans/IBMPlexSans-Italic.ttf");
 }
 
 /* Semi bold */
 @font-face {
-  font-family: "IBM Plex Sans", sans-serif;
+  font-family: "IBM Plex Sans";
   font-weight: 600;
   font-style: normal;
   font-display: var(--text-body-font-display, swap);
-  src: url("#{variables.$font-path}/IBMPlexSans/IBMPlexSans-SemiBold.ttf")
-    format("ttf");
+  src: url("#{variables.$font-path}/IBMPlexSans/IBMPlexSans-SemiBold.ttf");
 }
 
 @font-face {
-  font-family: "IBM Plex Sans", sans-serif;
+  font-family: "IBM Plex Sans";
   font-weight: 600;
   font-style: italic;
   font-display: var(--text-body-font-display, swap);
-  src: url("#{variables.$font-path}/IBMPlexSans/IBMPlexSans-SemiBoldItalic.ttf")
-    format("ttf");
+  src: url("#{variables.$font-path}/IBMPlexSans/IBMPlexSans-SemiBoldItalic.ttf") format("truetype");
```

Comment:
Is it correct that this one has `format("truetype")` and others do not have the format?

Reply:
My bad. `format("truetype")` is fine, it should work without, too, but `format("ttf")` might cause issues.